src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -222,6 +223,13 @@ protected override void AddResponseFileCommands(CommandLineBuilderExtension comm
             commandLine.AppendSwitchIfNotNull("/nullable:", Nullable);
             commandLine.AppendWhenTrue("/nosdkpath", _store, nameof(DisableSdkPath));
 
+            // Pass sdkpath if we are invoking core compiler from framework to preserve the behavior that framework compiler would have.
+            // Pass this option only to the built-in compiler (customer-supplied compiler might not support it).
+            if (IsSdkFrameworkToCoreBridgeTask && UsingBuiltinTool)
```

Comment:
Do we need a similar change for VB or did it not get broken with the bridge task change?